app/routes/questionnaire.py

```diff
@@ -283,6 +300,52 @@ def get_thank_you(schema):
     )
 
 
+@post_submission_blueprint.route("confirmation-email/send", methods=["GET", "POST"])
+@login_required
+def send_confirmation_email():
+    if not get_session_store().session_data.confirmation_email_sent:
+        raise NotFound
+
+    confirmation_email = ConfirmationEmail()
+
+    if request.method == "POST":
+        if confirmation_email.form.validate():
+            confirmation_email.handle_post()
+            return redirect(
+                url_for(
+                    "post_submission.get_confirmation_email_sent",
+                    email=confirmation_email.get_url_safe_serialized_email(),
+                )
+            )
+
+    return render_template(
+        template="confirmation-email",
+        content=confirmation_email.get_context(),
+        hide_signout_button=True,
+    )
+
+
+@post_submission_blueprint.route("confirmation-email/sent", methods=["GET"])
+@login_required
+def get_confirmation_email_sent():
```

Comment:
This page has the Exit button which when clicked results in a 405 as it is trying to `POST` when that is not allowed. It used to be a link which looked like a button, but it is now actually a button, so we may have to revisit that.